charts/argo-workflows/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: v3.6.2
+appVersion: v3.6.3
```

Comment:
Ah not this one but `version`  in line 6.

Reply:
Sorry, I mistaked and fixed it :bow: